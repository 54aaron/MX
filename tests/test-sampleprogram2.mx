int main() {
    Matrix m1;
    Matrix m2;

    m1 = [[1,2],[3,4]];
    m2 = [[2,4],[6,8]];

    print_matrix(m1 +. m2);
    print_matrix(m1 -. m2);
    print_matrix(m1 *. m2);
    print_matrix(m2 **. 3);
}
