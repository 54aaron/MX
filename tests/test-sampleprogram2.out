[ 3, 6 ]
[ 9, 12 ]

[ -1, -2 ]
[ -3, -4 ]

[ 14, 20 ]
[ 30, 44 ]

[ 6, 12 ]
[ 18, 24 ]

